packages/ra-ui-materialui/src/input/ReferenceArrayInput.stories.tsx

```diff
@@ -70,6 +70,68 @@ export const Basic = () => (
     </Admin>
 );
 
+export const ReadOnly = () => (
+    <Admin dataProvider={dataProvider} history={history}>
+        <Resource name="tags" recordRepresentation={'name'} />
+        <Resource
+            name="posts"
+            create={() => (
+                <Create
+                    resource="posts"
+                    record={{ tags_ids: [1, 3] }}
+                    sx={{ width: 600 }}
+                >
+                    <SimpleForm>
+                        <ReferenceArrayInput
+                            reference="tags"
+                            resource="posts"
+                            source="tags_ids"
+                            readOnly
+                        />
+                        <ReferenceArrayInput
+                            reference="authors"
+                            resource="name"
+                            source="authors_ids"
+                            readOnly
+                        />
+                    </SimpleForm>
+                </Create>
+            )}
+        />
+    </Admin>
+);
+
+export const Disabled = () => (
+    <Admin dataProvider={dataProvider} history={history}>
+        <Resource name="tags" recordRepresentation={'name'} />
+        <Resource
+            name="posts"
+            create={() => (
+                <Create
+                    resource="posts"
+                    record={{ tags_ids: [1, 3] }}
+                    sx={{ width: 600 }}
+                >
+                    <SimpleForm>
+                        <ReferenceArrayInput
+                            reference="tags"
+                            resource="posts"
+                            source="tags_ids"
+                            disabled
+                        />
+                        <ReferenceArrayInput
+                            reference="authors"
+                            resource="name"
+                            source="authors_ids"
+                            disabled
+                        />
```

Comment:
**suggestion**: there is no author on the record
```suggestion
                        
```

Reply:
It's voluntary. It's a request from JB: to be able to see input behavior with and without data.